tests/page-objects/reports/reports.wdio.page.js

```diff
@@ -110,9 +115,77 @@ const reportsListDetails = async () => {
   return reportDetails;
 };
 
+const collapseSelection = async () => {
+  await (await itemSummary()).click();
+  expect(await (await reportBodyDetails()).isExisting()).to.be.false;
+};
+
+const deleteSelectedReports = async (savedUuids) => {
+  const deleteAllButton = () => $('.action-container .detail-actions .delete-all');
+  const confirmButton = () => $('.btn.submit.btn-danger');
+
+  await (await deleteAllButton()).click();
+  await (await confirmButton()).click();
+  await (await firstReport ()).waitForDisplayed();
+  // make sure the reports are deleted
+  expect(await reportsByUUID(savedUuids[1]).length).to.equal(1);
+};
+
+const deselectAll = async () => {
+  const deselectAllButton = await $('.action-container .deselect-all');
+  await deselectAllButton.click();
+  const count = await $('#reports-content .selection-count > span');
+  expect(await count.isExisting()).to.be.false;
+  expect(await $$(reportBody).length).to.equal(0);
+};
+
+const expandSelection = async () => {
+  await (await itemSummary()).click();
+  await (await $(reportBodyDetailsSelector)).waitForDisplayed();
+};
+const selectAll = async () => {
+  await (await $('.action-container .select-all')).click();
+  await (await $('#reports-content .selection-count > span')).waitForDisplayed();
+  expect(await $$(reportBody).length).to.equal(3);
```

Comment:
I think that we should not use a magic number here to assert. We should either move this assertion in the main test file or pass the number as a parameter to the function and keep the assertion here.